src/components/rating/Rating.spec.jsx

```diff
@@ -95,12 +95,6 @@ describe('rating', () => {
 
     stars.at(1).simulate('click');
     expect(onChange.mock.calls).toHaveLength(2);
-
-    const lastRatedStarIndex = rate - 1;
```

Comment:
this caused error:
when current rate was === 4, then clicking on 4th star didn't call update ;(